packages/grpc-native-core/index.d.ts

```diff
@@ -941,15 +947,15 @@ declare module "grpc" {
      * @param callback The callback to for when the response is received
      * @return An event emitter for stream related events
      */
-    makeUnaryRequest(
+    makeUnaryRequest<T>(
```

Comment:
This again conflates the request and response types. The `argument` parameter should have the request type, and the `callback` parameter should have the response type. And it's not clear which of those is supposed to be the type parameter for `ClientUnaryCall`.

Also, if you're going to add type parameters to this function, it would make sense to do the same with `serialize` and `deserialize`.